docs/modules/ROOT/partials/storage-ceph-remove-mon.adoc

```diff
@@ -52,26 +67,31 @@ kubectl --as=cluster-admin -n syn-rook-ceph-operator logs -f \
   deploy/rook-ceph-operator
 ----
 
-. Cordon node to {mon-operation} and delete MON pod
+. Cordon old nodes
 +
 [source,bash,subs="attributes+"]
 ----
-kubectl --as=cluster-admin cordon "{mon-node}"
-kubectl --as=cluster-admin -n syn-rook-ceph-cluster delete pod \
-  -l app=rook-ceph-mon,ceph_daemon_id="${MON_ID}"
+for node in $(echo -n {mon-replace-list}); do
+  kubectl --as=cluster-admin cordon "${node}"
+done
 ----
-
-. Wait until new MON is scheduled
 +
-[source,bash]
-----
-kubectl --as=cluster-admin -n syn-rook-ceph-cluster get pods -w
-----
+. For every id in `$MON_IDS` replace the MON pod
 +
 [source,bash]
 ----
+mon_id=<MON_ID>
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster delete pod \
+  -l app=rook-ceph-mon,ceph_daemon_id="${mon_id}"
+
+# Wait until new MON is scheduled
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster get pods -w
+
+# Wait until the cluster has regained full quorum
 kubectl --as=cluster-admin -n syn-rook-ceph-cluster exec -it deploy/rook-ceph-tools -- \
   ceph status
+
+# Repeat for all other $MON_IDS
 ----
```

Comment:
Just a note: This seems a bit clunky for how-tos where we only replace a single MON, but I'm not sure how to improve it right now.

Reply:
I tried to automate the waiting before but it's a tad risky and fragile. I removed all MONs at once.